### PR TITLE
feat: implement IFE txbytes to id

### PR DESCRIPTION
**Pull request by unnawut**
Relates to https://github.com/omisego/elixir-omg/issues/1347

## Overview

Bring [plasma-contracts' `ExitID.getInFlightExitId()`](https://github.com/omisego/plasma-contracts/blob/a3acd36/plasma_framework/contracts/src/exits/utils/ExitId.sol#L53-L55) into ExPlasma.

### CODE REVIEW

**Comment by pdobacz** [comment]
Ah, commented on https://github.com/omisego/elixir-omg/issues/1347#issuecomment-590806609 just a second ago. Considering this is PRed already, I think the only relevant comment from there is:

> I think doing the calculation off-chain (that is, by a twin implementation in Elixir), a thorough conformance test would be needed to make sure both implementations are kept in sync. It should be a few lines of propcheck

WDYT?

**Comment by unnawut** [comment]
@pdobacz Addressed your conformance test suggestion!

File: docker/migrations/2_deploy_exit_id_wrapper.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+var Contract = artifacts.require("ExitIdWrapper");
```
Comment (pdobacz) [inline docker/migrations/2_deploy_exit_id_wrapper.js]: oh, I didn't know this existed, excellent!

File: test/conformance/in_flight_exit_test.exs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  @ife_tx_hexes [
+    "0x",
+    "0x1234",
+    "0xb26f143eb9e68e5b",
+    "0x70de28d3cd1cb609",
+    "0xc235a61a575eb3e2",
+    "0x8fdeb13e6acdc74955fdcf0f345ae57a",
+    "0x00000000000000000000000000000000",
+    "0xffffffffffffffffffffffffffffffff"
+  ]
```
Comment (achiurizo) [inline test/conformance/in_flight_exit_test.exs]: :100:

**Comment by unnawut** [comment]
Thanks all!

**Pull request merged**